packages/blocks/servicenow/src/actions/create-fields-properties.ts

```diff
@@ -0,0 +1,127 @@
+import { Property } from '@openops/blocks-framework';
+import { ServiceNowAuth } from '../lib/auth';
+import { createFieldValueProperty } from '../lib/create-field-value-property';
+import { getServiceNowTableFields } from '../lib/get-table-fields';
+
+/**
```

Comment:
please remove the comments, unless you think they are absolutely necessary to understanding the code